### PR TITLE
GITC-616: fetch gas price from polygon gas tracker

### PR DESCRIPTION
**Pull request by chibie**
##### Description

This pr fetches the gas price from polygon gas tracker as opposed to hardcoding it.

##### Testing

tested locally

### CODE REVIEW

File: app/assets/v2/js/cart-ethereum-polygon.js [hunk 1]
```diff
@@ -381,6 +382,14 @@ Vue.component('grantsCartEthereumPolygon', {
 
       let gasLimit = 0;
 
+      // fetch gas prices from polygon gas tracker
+      if (!this.polygon.gasPrices) {
+        const priceUrl = 'https://api.polygonscan.com/api?module=gastracker&action=gasoracle&apikey=I28K1DVQAWAISBSI146I71YQDBK6N1C9GJ';
```
Comment (gdixon) [inline app/assets/v2/js/cart-ethereum-polygon.js:387]: We might want to switch this out for a paid account or move this to the backend and periodically update it? This is going to get hammered if we call it every time a user hits the checkout.

Comment (gdixon) [inline app/assets/v2/js/cart-ethereum-polygon.js:387]: We might want to switch this out for a paid account or move this to the backend and set up a cron to store and update the `safeGasPrice` periodically.

This is going to get hammered if we call it every time a user hits the checkout.

Comment (thelostone-mc) [reply app/assets/v2/js/cart-ethereum-polygon.js:387]: I'd say set a high default price and wrap this in a try catch for now

Comment (nutrina) [reply app/assets/v2/js/cart-ethereum-polygon.js:387]: Shouldn't we load the api key from the settings?

Comment (thelostone-mc) [reply app/assets/v2/js/cart-ethereum-polygon.js:387]: Oh that is a nice catch ! @chibie i agree with nutrina here

Comment (gdixon) [reply app/assets/v2/js/cart-ethereum-polygon.js:387]: Yeah we should really be protecting that key - I would recommend moving this to the backend and returning the `safeGasPrice` as part of the context.

Comment (chibie) [reply app/assets/v2/js/cart-ethereum-polygon.js]: @gdixon pushed a commit https://github.com/gitcoinco/web/pull/9733/commits/c2cfb70b9e1bb1f0a8bd3a58ff70de63c1f30aab

we should run `./manage.py fetch_gas_prices` after deploy

**Review by gdixon — APPROVED** [review]
Looks great @chibie, nicely done 🙌

**Review by thelostone-mc — APPROVED** [review]

**Pull request merged**